include/common_structs.h

```diff
@@ -558,9 +557,9 @@ typedef struct Camera {
     /* 0x03A */ char unk_3A[2];
     /* 0x03C */ Vec3f lookAt_eye;
     /* 0x048 */ Vec3f lookAt_obj;
-    /* 0x054 */ f32 unk_54;
-    /* 0x058 */ f32 unk_58;
-    /* 0x05C */ f32 unk_5C;
+    /* 0x054 */ f32 unk_54; // x-related
+    /* 0x058 */ f32 unk_58; // y-related
+    /* 0x05C */ f32 unk_5C; // z-related
```

Comment:
We should make this a Vec3f unk_54 perhaps?

Reply:
I'd like to wait until we know what it is and that they're always related / grouped